src/navigator-card-body.jsx

```diff
@@ -345,6 +271,8 @@ export const NavigatorCardBody = ({
             <>
                 {contextMenu}
                 <ListingTable
+                  onClick={resetSelected}
+                  onContextMenu={resetSelected}
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-307-20240307-112122-d90b1ce5-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)